### PR TITLE
Big ol Soda Update/Vending Resprite 1

### PR DESCRIPTION
**Pull request by scarwolff**
## Description

Coming at yah with 3 new fallout themed sodas! And appropriate vending machines randomised for each drink, replaces the current soda machines so no need for mappers to do anything!
## Motivation and Context

Sick of seeing pwr-game in a fallout game, And no Sarsaparilla? Wack.
## How Has This Been Tested?

It runs.
## Screenshots (if appropriate):
image
image

## Changelog (necessary)
:cl:
add: Added New Sodas and Sprites
add: Resprited Coffee/Smokes machine and Nuka-cola
del: Removed cringe ss13 sodas
/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Got crushed sprites?

**Comment by scarwolff** [comment]
this where im stumped man how do you and why can you crush a bottle on your head, original nuka didnt have one so these dont either

**Comment by Laus4Deus** [comment]
Yeah!
crushing them bottles is more an oversight.

**Comment by Laus4Deus** [comment]
The code part seems very clean.
Love it!

File: code/modules/vending/_vending.dm [hunk 1]
```diff
@@ -421,6 +421,7 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 		use_power(5)
 		if(icon_vend) //Show the vending animation if needed
 			flick(icon_vend,src)
+		playsound(src, 'sound/machines/machine_vend.ogg', 50, TRUE, extrarange = -3)
```
Comment (Laus4Deus) [inline code/modules/vending/_vending.dm:424]: What is Extrarange?

Comment (scarwolff) [reply code/modules/vending/_vending.dm:424]: shrinks other being able to hear the noise from the default distance by 3 tiles, didnt make sense for someone a 9 or so tiles away to here the kerklunk

**Pull request merged**